clinica/iotools/bids_utils.py

```diff
@@ -160,6 +161,7 @@ def create_sessions_dict(
         bids_ids: list of bids ids
         name_column_ids: name of the column where the subject ids are stored
         subj_to_remove: subjects to remove
+        participants_df: a pandas dataframe that contains the participants data (required for OASIS3 only)
```

Comment:
As discussed, I updated the create_sessions_dict in another PR so that it is used by OASIS and OASIS3 only. Please update the code accordingly